packages/code-du-travail-frontend/package.json

```diff
@@ -73,7 +73,8 @@
     "styled-components": "^5.1.1",
     "unified": "^9.0.0",
     "unist-util-find": "^1.0.1",
-    "url-search-params": "^1.1.0"
+    "url-search-params": "^1.1.0",
+    "uuid": "^8.2.0"
```

Comment:
can we use a lighter library like 
- https://bundlephobia.com/result?p=uuid-random@1.3.2
- https://bundlephobia.com/result?p=cuid@2.1.8
- https://bundlephobia.com/result?p=nanoid@3.1.10
 vs 
- https://bundlephobia.com/result?p=uuid@8.2.0

Reply:
nooormalement uuid v4 ne pèse même pas un Ko ?

<img width="298" alt="Screenshot 2020-07-09 at 14 19 59" src="https://user-images.githubusercontent.com/705453/87039415-68ee1780-c1ef-11ea-8786-2c1b39d75844.png">